.github/workflows/test-EMULATOR.yml

```diff
@@ -3,14 +3,27 @@ name: OSW-EMULATOR-test
 on:
   workflow_dispatch:
   push:
+    paths_ignore: '["**/*.md", "**/scripts/screen_capture/**"]'
```

Comment:
I think the screen capture path is somewhat very specific - but I have no idea to make it better :)